metrics/package.go

```diff
@@ -14,15 +14,29 @@ func Mark(name string) {
 
 // TimeSince increases the timer metric with the given name by the time since the given time
 func TimeSince(name string, since time.Time) {
+	metrics.DefaultRegistry.Register(name, newTimerWithSmallerSampleSize)
```

Comment:
I think `Register` will cause an overwrite. Should probably be `RegisterOrGet` instead?